src/lsptoolshost/razorCommands.ts

```diff
@@ -111,6 +112,24 @@ export function registerRazorCommands(context: vscode.ExtensionContext, language
         })
     );
 
+    const semanticTokensRangesRequestType = new RequestType<
+        SerializableSemanticTokensParams,
+        vscode.SemanticTokens,
+        any
+    >('roslyn/semanticTokenRanges');
```

Comment:
I think this is the new endpoint for the feature flag, so we probably don't want to use it. We should be able to use the standard `textDocument/semanticTokens/range` method.

Reply:
That's fair. It is kind of weird since we're still being sent an array of ranges from the server, so I had to hardcode in `ranges[0]` to use the standard method.

Also, since we're going back to the standard method, I tested out the built-in `'vscode.provideDocumentRangeSemanticTokens` command and was actually able to get it working this time around. 😄 Definitely simplified the code